iFixit/react-commerce
# Add children section heading override field

fixes #232 
 
## Changelog

- Add strapi children heading field
- Use device title instead of product list title
- Update product list seed

## QA

1. Visit the Vercel PR preview (replace vercel.app with cominor.com) 
2. Visit a few product lists and verify that the title above the children subcategories is in the shape `Choose model of <DEVICE TITLE>`
3. Login into [Strapi for this branch](https://improve-product-list-subcategories-heading.govinor.com/admin/) and select a product list. Use the children heading text field to customize the section title and verify that now the product list children section heading reflect what you've set in Strapi.
